i18n/errors/en.yml

```diff
@@ -1,10 +1,32 @@
 # Error definitions, usage Text.e.ERR999
+#
 # General format:
+# ERRORID
+#   display: optional hash of display options. See display_defaults below.
+#   text: |
```

Comment:
Instead of having to add a `text` subfield to all errors (when I am assuming most will inherit the default options) could we instead do a check when we fetch the error to see if it has a `display` subfield? And if not just use the default options and assume the contents are the text?

Reply:
Yes, though I'll do that in a subsequent PR to prevent a cascade of ick.

Reply:
Offline discussion landed on leaving this as-is (needing .text) to keep usage consistent for all errors.